frameworks/angular/container/package.json

```diff
@@ -20,8 +21,8 @@
     "@angular/platform-browser-dynamic": "^18.2.3",
     "@angular/router": "^18.2.3",
     "@finos/fdc3": "2.0.3",
-    "@openfin/core": "38.82.69",
-    "@openfin/workspace": "19.0.4",
+    "@openfin/core": "38.82.64",
+    "@openfin/workspace": "19.1.20",
```

Comment:
Just a small thing. This should still be workspace v19 not 19.1 as it is going into main/v19 branch.

Reply:
Reverted to 19.0.4